### ISSUE TITLE
Payment amount options (partial payments)

### ISSUE THREAD

**Issue by clarkwmcd** [snarfblam/Prop-manager#68]
Allow the option for users to pay less then the full rent amount. For example if two users are sharing a unit they should be able to each pay half

**Comment by snarfblam** [comment]
Hmm... I suppose making a partial payment record would update the price of the original, mark it as paid, and generate a new payment record for the remainder?

**Comment by clarkwmcd** [comment]
Yeah that would make sense. I forgot there was a column for "paid" with a boolean. 

Could also just add another column with the running balance. The "amount" column would have the month rent charge and the new column would be the balance left
